examples/typescript/alexa/lists/src/components/GlobalComponent.ts

```diff
@@ -0,0 +1,66 @@
+import { BaseComponent, Component, Global, Handle, Jovo } from '@jovotech/framework';
+import { LoveHatePizzaComponent } from './LoveHatePizzaComponent';
+
+@Global()
+@Component()
+export class GlobalComponent extends BaseComponent {
+  LAUNCH() {
+    return this.$redirect(LoveHatePizzaComponent);
+  }
+
+  // ITEMS CREATED
+  @Handle({
+    global: true,
+    types: ['AlexaHouseholdListEvent.ItemsCreated'],
+    platforms: ['alexa'],
+  })
+  handleCreatedItems() {
+    const { listId, listItemIds } = getIdsFromRequest(this);
+    console.log(`Added ${listItemIds} to ${listId}`);
+
+    if (!listItemIds || !listId) {
+      return;
+    }
+    Promise.all(listItemIds.map(itemId => this.$alexa?.$user?.getListItem(listId, itemId)))
+        .then(result => console.log('The created items are: ', result))
+  }
+
+  // ITEMS UPDATED
+  @Handle({
+    global: true,
+    types: ['AlexaHouseholdListEvent.ItemsUpdated'],
+    platforms: ['alexa'],
+  })
+  async handleUpdatedItems() {
+    const { listId, listItemIds } = getIdsFromRequest(this);
+    console.log(`Modified ${listItemIds} from ${listId}`);
+  }
+
+  // ITEMS DELETED
+  @Handle({
+    global: true,
+    types: ['AlexaHouseholdListEvent.ItemsDeleted'],
+    platforms: ['alexa'],
+  })
+  async handleDeletedItems() {
+    const { listId, listItemIds } = getIdsFromRequest(this);
+    console.log(`Deleted ${listItemIds} from ${listId}`);
+  }
+}
+
+type ListItemRequest = {
+  listId?: string;
+  listItemIds?: string[];
+};
+
+function getIdsFromRequest(jovo: Jovo): ListItemRequest {
```

Comment:
Thanks for your PR @NLueg! Looks great.

Just one question about this ´getIdsFromRequest` helper: Is this maybe something that could be added as a method, either to the `ListApi` or the `AlexaRequest` class? Ideally, I'd like to keep handlers and utility functions as separate as possible.

Reply:
Jep, this totally makes sense. Moved the function to the `ListApi`!

Reply:
Thank you @NLueg! After going through the Alexa docs and the code a bit more and discussing it with @aswetlow, I decided to take out the helper method because it's one line of code in the handler. Apologies for the stress!